pr_check.sh

```diff
@@ -12,26 +12,38 @@ export APP_ROOT=$(pwd)
 export NODE_BUILD_VERSION=16
 COMMON_BUILDER=https://raw.githubusercontent.com/RedHatInsights/insights-frontend-builder-common/master
 
+set -exv
+# source is preferred to | bash -s in this case to avoid a subshell
+source <(curl -sSL $COMMON_BUILDER/src/frontend-build.sh)
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+# shellcheck source=/dev/null
+curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
+
 # --------------------------------------------
 # Options that must be configured by app owner
 # --------------------------------------------
-# IQE tests for patchman ui is not set yet. We can comment out config for it for now
-# IQE_PLUGINS=""
-# IQE_MARKER_EXPRESSION=""
-# IQE_FILTER_EXPRESSION=""
+export DEPLOY_FRONTENDS="true"
+export IQE_ENV="ephemeral"
+export IQE_SELENIUM="true"
+export APP_NAME="patchman"
+export DEPLOY_TIMEOUT="900"  # 15min
+export IQE_CJI_TIMEOUT="30m"
+export REF_ENV="insights-stage"
+export COMPONENTS_W_RESOURCES="vmaas"
```

Comment:
I think you need to combine these to be: `export COMPONENTS_W_RESOURCES="vmaas patchman"` otherwise the vmaas one is overwritten. See here:  https://github.com/RedHatInsights/vulnerability-engine/blob/master/pr_check.sh#L9C19-L9C19

Reply:
@mtclinton thanks for looking into it. i tried your above suggestion. Vmaas works fine.
raised another PR https://github.com/RedHatInsights/patchman-ui/pull/1077
now vmaas looks good to me :)